devices/devices_unix.go

```diff
@@ -14,42 +16,43 @@ func DeviceInfo(fi os.FileInfo) (uint64, uint64, error) {
 		return 0, 0, fmt.Errorf("cannot extract device from os.FileInfo")
 	}
 
-	return getmajor(sys.Rdev), getminor(sys.Rdev), nil
+	dev := uint64(sys.Rdev)
```

Comment:
`sys.Rdev` is already uint64, maybe just pass in directly to the functions

Reply:
e.g. Darwin uses `int32`, FreeBSD uses `uint32`. Only Linux already uses `uint64` AFAICS. Thus, the type conversion is necessary.